### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GurpreetSingh31**
Just added the Docker Desktop and some Resources

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7026"></a>

### CODE REVIEW

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
I think having links to resources is good, but they are not a `prerequisite` to run the app

I suggest moving to a `resources` section

File: README.md
Comment (CBID2) [inline README.md:83]: [Proofreading] I think adding 'V2' after the name of the docker would make it easier for contributors to understand which version you're referring to. 
```suggestion
- [Docker Compose V2](https://github.com/docker/compose) V2
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @GurpreetSingh31! :) I pointed out a proofreading error that could in my inline comment.

File: README.md
Comment (SaraJaoude) [reply README.md:83]: This results in there being x2 `V2` as shown on the screenshot so this will not work. 
Screenshot 2023-05-24 at 10 57 55

**Comment by SaraJaoude** [comment]
> Hi @GurpreetSingh31! :) I pointed out a proofreading error that could in my inline comment. Fix it and you're good to go! :)

@CBID2 we appreciate your enthusiasm to get this PR merged but as has been mentioned to you in the past by other maintainers, please refrain from saying that once your change is accepted the PR is "good to go" before it is actually reviewed and approved by one of the maintainers (as this causes confusion to the PR author).

**Comment by CBID2** [comment]
> > Hi @GurpreetSingh31! :) I pointed out a proofreading error that could in my inline comment. Fix it and you're good to go! :)
> 
> 
> 
> @CBID2 we appreciate your enthusiasm to get this PR merged but as has been mentioned to you in the past by other maintainers, please refrain from saying that once your change is accepted the PR is "good to go" before it is actually reviewed and approved by one of the maintainers (as this causes confusion to the PR author).

Noted

**Comment by CBID2** [comment]
> > Hi @GurpreetSingh31! :) I pointed out a proofreading error that could in my inline comment. Fix it and you're good to go! :)
> 
> 
> 
> @CBID2 we appreciate your enthusiasm to get this PR merged but as has been mentioned to you in the past by other maintainers, please refrain from saying that once your change is accepted the PR is "good to go" before it is actually reviewed and approved by one of the maintainers (as this causes confusion to the PR author).

Just edited my comment

**Comment by SaraJaoude** [comment]
Due to lack of activity I will be closing this Pull Request.

**Pull request closed without merging**
